salt/metalk8s/defaults.yaml

```diff
@@ -39,29 +39,13 @@ repo:
   port: 8080
   registry_endpoint: 'metalk8s-registry-from-config.invalid'
   repositories:
-    metalk8s-base:
-      humanname: CentOS - Base
-      gpgcheck: 1
-      gpgkeys:
-        - RPM-GPG-KEY-metalk8s-base-7_1
-      repo_gpg_check: 0
-      enabled: 0
-
     metalk8s-epel:
       humanname: EPEL
       gpgcheck: 1
       gpgkeys:
         - RPM-GPG-KEY-metalk8s-epel-7_1
       repo_gpg_check: 0
-      enabled: 0
-
-    metalk8s-extras:
-      humanname: CentOS - Extras
-      gpgcheck: 1
-      gpgkeys:
-        - RPM-GPG-KEY-metalk8s-extras-7_1
-      repo_gpg_check: 0
-      enabled: 0
+      enabled: 1
```

Comment:
Any reason to enable these now?

Reply:
Yes, because pkg.installed.fromrepo disable everything and enable only the repositories provided (metalk8s-*).
So we can't disable our repositories and rely on it to enable our repos, otherwise we can't get dependencies from system repositories.

Maybe we could enable and disable metalk8s repos before and after each salt execution (but mêêêh).
The other thing we could do is setting a list of repositories (with base, updates, extras & metalk8s repos) somewhere in the pillar/defaults/map we want to use with fromrepo... not sure which one is the good approach.

Or... we can do our own pkg.installed module...

Reply:
In essence, I'm not too fond of 'enabling our repos by default'.

Reply:
> In essence, I'm not too fond of 'enabling our repos by default'.

Me neither, but we're doing the same on Ubuntu.